lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1788,6 +1788,28 @@ defmodule Kernel.SpecialForms do
       end
       #=> "Will match"
 
+
+  ## Multiple match options
+
+  It's possible to match against several values using standard pattern matching syntax:
+
+      case data do
+        value when value in [:one, :two] ->
+          "#{value} has been matched"
+        :three ->
+          "three has been matched"
+      end
+
+  The case above will get compiled as follows:
```

Comment:
Hi @achempion!

It is not true the case above will be compiled to the case below but rather this:

```
      case data do
        value when value == :one or value == :two ->
          "#{value} has been matched"
        :three ->
          "three has been matched"
      end
```

Therefore I am a bit worried about documenting this as multiple matches because it isn't really multiple matches just a subset of guards.

Thoughts?

Reply:
The reason I choose this wording is the use case user will search for is how to write multiple matches in a case statement.

I'm not sure about including the last section how this code will unfold after compilation but I found it might be helpful.

Reply:
Maybe "Using guards to match against multiple values"?

Reply:
@achempion that works better, can you please amend! thank you.